addons/website_hr_recruitment/models/hr_job.py

```diff
@@ -35,14 +35,6 @@ def _get_default_website_description(self):
             <h6>4 Days after Interview</h6>
         """)
 
-    @api.onchange('website_published')
-    def _onchange_website_published(self):
```

Comment:
If someone has customized this _onchange_website_published then it will fail so we don't remove it in stable version.

//TODO: Remove in master
`@api.onchange('website_published')`
     `     def _onchange_website_published(self):`
                 `       pass `
cc @kba-odoo @sofiagvaladze